AttitudeManager/FlightModes/Inc/fbwa.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef ZPSW3_AM_FBWA_HPP
+#define ZPSW3_AM_FBWA_HPP
+
+#include "flightmode.hpp"
+
+namespace AM {
+
+class FBWA : public Flightmode {
+   public:
+    FBWA() = default;
+
+    AttitudeManagerInput run(const AttitudeManagerInput& input, AxisPIDs _pids) override;
```

Comment:
our PID's should exist as private class variables, and not have to be passed into the "run" function.

They can be set during the initialization of the class! (in the constructor).